src/inference/src/dev/isync_infer_request.cpp

```diff
@@ -143,8 +143,7 @@ ov::ISyncInferRequest::FoundPort ov::ISyncInferRequest::find_port(const ov::Outp
     ov::ISyncInferRequest::FoundPort::Type type = ov::ISyncInferRequest::FoundPort::Type::INPUT;
     for (const auto& ports : {get_inputs(), get_outputs()}) {
         for (size_t i = 0; i < ports.size(); i++) {
-            if (ports[i].get_index() == port.get_index() && ports[i].get_names() == port.get_names() &&
-                check_nodes(ports[i].get_node(), port.get_node())) {
+            if (ports[i].get_index() == port.get_index() && check_nodes(ports[i].get_node(), port.get_node())) {
```

Comment:
do we have a list of transformations where i/o tensor names are changed?

Reply:
Not yet. For this model "text_encoder" from Stable Diffusion pipeline in my reproduction, the transformation, [`EliminateReshape`](https://github.com/openvinotoolkit/openvino/blob/master/src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp#L331), changed the input tensor names.

Reply:
could you please create a bug for this transformation?

Reply:
Sure.

Reply:
133925

Reply:
maybe for now to fix the issue in jira, we can change the port name "==" to logic like this: the compiled model port names should include the query port names, I suppose this is the minimum impact to existing code logic.

Reply:
> we can change the port name "==" to logic like this: the compiled model port names should include the query port names

Agree with it as more concervative solution

Reply:
Updated.